{"variants":[{"paths":["\/documentation\/gtmappauth\/keychainstore\/error\/customnserror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"identifier":{"url":"doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/KeychainStore\/Error\/CustomNSError-Implementations","interfaceLanguage":"swift"},"topicSections":[{"title":"Instance Properties","identifiers":["doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/KeychainStore\/Error\/errorCode-82syd","doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/KeychainStore\/Error\/errorUserInfo-5l2v8"],"generated":true},{"title":"Type Properties","identifiers":["doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/KeychainStore\/Error\/errorDomain-2fnw1"],"generated":true}],"kind":"article","metadata":{"modules":[{"name":"GTMAppAuth"}],"role":"collectionGroup","title":"CustomNSError Implementations"},"hierarchy":{"paths":[["doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth","doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/KeychainStore","doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/KeychainStore\/Error"]]},"references":{"doc://GTMAppAuth/documentation/GTMAppAuth/KeychainStore/Error/errorCode-82syd":{"role":"symbol","title":"errorCode","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorCode"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Int","preciseIdentifier":"s:Si"}],"abstract":[],"identifier":"doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/KeychainStore\/Error\/errorCode-82syd","kind":"symbol","type":"topic","url":"\/documentation\/gtmappauth\/keychainstore\/error\/errorcode-82syd"},"doc://GTMAppAuth/documentation/GTMAppAuth/KeychainStore":{"role":"symbol","title":"KeychainStore","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"KeychainStore"}],"abstract":[{"type":"text","text":"A helper providing a concrete implementation for saving and loading auth data to the keychain."}],"identifier":"doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/KeychainStore","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"KeychainStore"}],"url":"\/documentation\/gtmappauth\/keychainstore"},"doc://GTMAppAuth/documentation/GTMAppAuth/KeychainStore/Error/errorDomain-2fnw1":{"role":"symbol","title":"errorDomain","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDomain"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"abstract":[],"identifier":"doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/KeychainStore\/Error\/errorDomain-2fnw1","kind":"symbol","type":"topic","url":"\/documentation\/gtmappauth\/keychainstore\/error\/errordomain-2fnw1"},"doc://GTMAppAuth/documentation/GTMAppAuth":{"role":"collection","title":"GTMAppAuth","abstract":[],"identifier":"doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth","kind":"symbol","type":"topic","url":"\/documentation\/gtmappauth"},"doc://GTMAppAuth/documentation/GTMAppAuth/KeychainStore/Error":{"role":"symbol","title":"KeychainStore.Error","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"Error"}],"abstract":[{"type":"text","text":"Errors that may arise while saving, reading, and removing passwords from the Keychain."}],"identifier":"doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/KeychainStore\/Error","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"Error"}],"url":"\/documentation\/gtmappauth\/keychainstore\/error"},"doc://GTMAppAuth/documentation/GTMAppAuth/KeychainStore/Error/errorUserInfo-5l2v8":{"role":"symbol","title":"errorUserInfo","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorUserInfo"},{"kind":"text","text":": ["},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":" : "},{"kind":"keyword","text":"Any"},{"kind":"text","text":"]"}],"abstract":[],"identifier":"doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/KeychainStore\/Error\/errorUserInfo-5l2v8","kind":"symbol","type":"topic","url":"\/documentation\/gtmappauth\/keychainstore\/error\/erroruserinfo-5l2v8"}},"variantOverrides":[{"patch":[{"op":"replace","path":"\/references\/doc:~1~1GTMAppAuth~1documentation~1GTMAppAuth~1KeychainStore\/title","value":"GTMKeychainStore"},{"op":"replace","path":"\/references\/doc:~1~1GTMAppAuth~1documentation~1GTMAppAuth~1KeychainStore\/fragments","value":[{"kind":"identifier","text":"GTMKeychainStore"}]}],"traits":[{"interfaceLanguage":"occ"}]}]}